Samples/HelloWorld.NetCore/src/OrleansClient/OrleansClient.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Orleans.Core" Version="$(OrleansPackageVersion)" />
+    <PackageReference Include="Microsoft.Orleans.Core" Version="2.0.0-rc1" />
```

Comment:
In the past we were just updating the value of OrleansPackageVersion in Directory.Build.props

Reply:
I think we should continue doing that for now

Reply:
but that would update every sample to use that version, which indicates we update all samples at once? or am I misunderstanding anything

Reply:
The issue is that the value of `OrleansPackageVersion` is currently `2.0.0-dev`, so it wouldn't work, correct?

I think we should have a Directory.Build.props dedicated for samples

Reply:
Clarification: I think we should have one Directory.Build.props per sample. Otherwise we will break all samples when updating it